src/DesktopVirtualization/README.md

```diff
@@ -99,4 +99,14 @@ directive:
       parameter-name: Force
     set:
       parameter-description: 'Specify to force userSession deletion.'
+  - where:
+      model-name: ScalingPlanPersonalSchedule
+      property-name: RampUpAutoStartHost
+    set:
+      property-name: RampUpAutoStartHosts
+  - where:
+      subject: ScalingPlanPersonalSchedule
+      parameter-name: RampUpAutoStartHost
+    set:
+      parameter-name: RampUpAutoStartHosts
```

Comment:
This config for "RampUpAutoStartHosts" causes the failure of CI. The parameter name should be singular. Pelase remove the renaming here.